build.sbt

```diff
@@ -104,8 +112,12 @@ lazy val libats_auth = (project in file("libats-auth"))
   .settings(Publish.settings)
   .dependsOn(libats)
 
-lazy val libats_root = (project in file("."))
+val metrics_root = project
```

Comment:
And removed in https://github.com/advancedtelematic/libats/commit/46314a4c60addcce9b7c5eda25f5c78305604b32.